v1/lib/consoleOutput.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * @file
+ * @version 0.1
+ * @copyright 2016 CN-Consult GmbH
+ * @author Max Späth <max.spaeth@cn-consult.eu>
+ */
+
+require_once 'baseoutput.php';
+
+/**
+ * This class implements the output function from BaseOutput and prints the gamefield every cycle
+ * to the console.
```

Comment:
This class inherits from the BaseOutput class, so we need to implement the output() function here, which will print every game cycle the complete gamefield on a console.